### PR TITLE
keymaps: add kinesis/keymaps/stapelberg

### PR DESCRIPTION
**Pull request by stapelberg**
## Description

Created using `qmk new-keymap -kb kinesis/kint36` and adjusted as described in the `readme.md`.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by stapelberg** [comment]
This should be ready to merge? :)

File: keyboards/kinesis/keymaps/stapelberg/config.h [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#pragma once
```
[the remark this answers is not in the document]
Comment (stapelberg) [reply keyboards/kinesis/keymaps/stapelberg/config.h:1]: Done

File: keyboards/kinesis/keymaps/stapelberg/keymap.c [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  return true;
+}
+
+void led_set_user(uint8_t usb_led) {
+
+}
```
[the remark this answers is not in the document]
Comment (stapelberg) [reply keyboards/kinesis/keymaps/stapelberg/keymap.c:69]: Done

File: keyboards/kinesis/keymaps/stapelberg/rules.mk [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+
```
[the remark this answers is not in the document]
Comment (stapelberg) [reply keyboards/kinesis/keymaps/stapelberg/rules.mk]: Done

File: keyboards/kinesis/keymaps/stapelberg/rules.mk [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+
+# Overwrite/amend what keyboards/kinesis/rules.mk defines:
+
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (fauxpark) [inline keyboards/kinesis/keymaps/stapelberg/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = no
COMMAND_ENABLE = yes
NKRO_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kinesis/keymaps/stapelberg/rules.mk (hunk 4 above)
Comment (stapelberg) [reply keyboards/kinesis/keymaps/stapelberg/rules.mk:10]: Done

**Review by fauxpark — APPROVED** [review]

**Comment by stapelberg** [comment]
This should be ready to merge, I think :)

File: keyboards/kinesis/keymaps/stapelberg/keymap.c [hunk 5]
```diff
@@ -0,0 +1,53 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/kinesis/keymaps/stapelberg/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Comment (stapelberg) [reply keyboards/kinesis/keymaps/stapelberg/keymap.c:1]: Done

**Comment by stapelberg** [comment]
This should be ready to merge

**Comment by stapelberg** [comment]
This should be good to merge now

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**